CONTRIBUTING.md

```diff
@@ -59,6 +67,22 @@ need help on it or would like suggestions on how to approach the issue. If so,
 share wildly. If they seem to have a good handle on it, let them work on their
 solution until a challenge comes up.
 
+#### Tips
+
+- At any point you can compare your feature branch to the upstream/main of
+  `LAION-AI/Open-Assistant` but using a URL like this:
+  https://github.com/LAION-AI/Open-Assistant/compare/main...andrewm4894:Open-Assistant:my-example-feature-branch.
```

Comment:
Maybe make this an env variable: `${MY_GITHUB_HANDLE)

Reply:
I kinda wanted to have a link they could click and see something so they can see a real example and what to expect and then just adjust the url for their own use case. 

actually might try get best of both worlds by updating the text below.